examples/access-token-server/package.json

```diff
@@ -28,11 +28,16 @@
     "@types/node": "^14.11.2",
     "@types/node-fetch": "^3.0.3",
     "jest": "^26.4.2",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "^2.6.7",
     "nodemon": "^2.0.4",
     "test": "^0.6.0",
     "ts-jest": "^26.5.6",
     "ts-node": "^10.2.1",
     "typescript": "^4.4.3"
+  },
+  "resolutions": {
+    "**/node-fetch": "2.6.7",
```

Comment:
Is node-fetch needed here?

Reply:
yes https://github.com/looker-open-source/sdk-codegen/security/dependabot/21

Reply:
Sorry, should have been clearer. node-fetch ^2.6.7 is included as a direct dependency in package.json so I was wondering if this is needed. Will approve anyway